plugins/snapshots/windows/cimfs.go

```diff
@@ -426,3 +424,55 @@ func createScratchVHDs(ctx context.Context, path string) (err error) {
 	}
 	return nil
 }
+
+const (
+	// LayerCimPathFlag is the option flag used to represent the path at which a layer CIM must be stored. This
+	// flag is only included if an image layer is being extracted onto the snapshot i.e the snapshot key has an
```

Comment:
This statement doesn't seem to be true, it looks like we always set LayerCimPathFlag on the mount?